src/Jackett.Common/Content/custom.js

```diff
@@ -1352,6 +1366,20 @@ function updateSearchResultTable(element, results) {
                     table.api().draw();
                 });
                 deadfiltercheckbox.prop('checked', settings.deadfilter);
+
+                savepresetlabel = $('<button id="jackett-search-results-datatable_savepreset_button" title="Save Preset" class="btn btn-success btn-sm" style="margin-left: 10px;"><span class="fa fa-plus"></span></button>');
```

Comment:
```suggestion
                savepresetlabel = $('<button id="jackett-search-results-datatable_savepreset_button" title="Save Search Preset" class="btn btn-success btn-sm" style="margin-left: 10px;"><span class="fa fa-plus"></span></button>');
```

Reply:
1. I was first trying for the right-hand side but the search box is a part of the table itself so had some trouble doing so. I'm very bad at UI but will give it another try.
2. Go to [Font Awesome -> Search](https://fontawesome.com/search) where you can find your desired icon and lmk which one you decided.
3. I am thinking that once you select a preset, the green button turns red and a different icon. Clicking on that will remove the selected preset. What do you think @ilike2burnthing ?

Reply:
1. if it's not workable that's fine, the modified tooltip should clear up any confusion
2. https://fontawesome.com/search?q=save&o=r&f=classic
![save](https://user-images.githubusercontent.com/59480337/193474502-29ad5589-d45b-4ec4-8002-95148653005e.png)
any of those really, just whatever you think looks best
3. either the same icon (top row) and then change the button colour, or use the arrow (second row) for the green button and the X (bottom row) for the red button

Reply:
Nice selection. Will see what I can do.

Reply:
1. I was right, Search box is part of the table so shifting the button to right means after the table (bottom). I suppose some CSS might be able to fix this, but UI isn't exactly my forte.
2. 2nd and 3rd row contains icons which are Pro versions so can't really use them.
3. The repository is using a really old version of FontAwesome i.e. 2 major versions back (v4 instead of v6 ). This means only basic icons are allowed. Upgrading that should be a separate PR I think because I tried to fiddle with it a little bit but couldn't get it working.

Reply:
@garfield69's commit is working great, just need to be able to remove individual presets now. I also discovered that you can add multiple versions of the same preset, but if we can remove them then that isn't an issue.

Reply:
Yup, that commit worked like a charm.
Now, I've included a way to remove a preset as well as prevent duplicate presets from being saved.

Reply:
HYPED! 🔥